docs/migration.adoc

```diff
@@ -3,4 +3,4 @@
 
 include::migration/migration110to120.adoc[leveloffset=+1]
 include::migration/migration120to130.adoc[leveloffset=+1]
-include::migration/migration130to140.adoc[leveloffset=+1]
+include::migration/migration130to140.adoc[leveloffset=+1]
```

Comment:
gyanus hogy nem valtozott semmi es megis...

Reply:
Ezt én sem teljesen értem, az eredetit raktam be a magyarnak is és angolnak is (természetesen lefordítva)